eva_submission/nextflow/remap_and_cluster.nf

```diff
@@ -281,3 +280,25 @@ process backpropagate_clusters {
     """
 }
 
+workflow {
+    main:
+        species_name = params.species_name.toLowerCase().replace(" ", "_")
+
+        // Create an channel that will either be empty if remapping will take place or contain a dummy value if not
```

Comment:
I changed it to use If statement in the workflow which makes it easier to read. I had to keep most of the when statement because they apply to each analysis rather than the whole remapping.

Reply:
Makes sense, thanks!